src/api/index.ts

```diff
@@ -20,7 +22,22 @@ const getLocalStorage: GetLocalStorage = (key, errorMsg = 'Something went wrong'
   }
 }
 
-export const requestNotes = () => new Promise(getLocalStorage('notes'))
+type GetUserNotes = () => (resolve: PromiseCallbackFun, reject: PromiseCallbackFun) => void
```

Comment:
nit: I know this pattern is used above, it feels pedantic to not just define the parameter types in the below function declaration